src/main/java/frc/robot/RobotContainer.java

```diff
@@ -54,6 +61,6 @@ private void configureButtonBindings() {
    */
   public Command getAutonomousCommand() {
     // An ExampleCommand will run in autonomous
-    return m_autoCommand;
+    return mDefaultDriveCommand;
```

Comment:
For the moment we should probably replace this with 'null' or an empty command object.